### ISSUE TITLE
[EN][Cata][Dungeon] The Vortex Pinnacle (Normal)

### ISSUE THREAD

**Issue by Illusie** [NRG-Wow/NRGWow-Bugtracker#67]
[//]: #  (Only reports in english will be allowed. and specific issues. everything like "ZOMG NOTHING WORKS" will be ignored / deleted)

**Reporter(Player/Character) Name**: ----

**Description**: 

**First impression of The Vortex Pinnacle as I experienced it in other Cata servers.
Never played retail so do not take my issue as the 'complete truth' thank you.**

1. 1st boss platform / aggro issue:
when you enter his platform you immediately get into fight(combat)
according to other servers, and yt video I watched you should enter into combat only after you pass the first ring(s) on ground.

2. 1st boss little tornados & light chains. (aoe damage):
when you pass them they should stop hit you with Light chain, they basically don't and keep attack you from inside.

3. Little circles on ground to click (idk what they are) & Slipstream:

Those little clickable circles should not even exist, never seen them on any Cata server + checked on YT videos. about the Slipstream it is very slow should be faster & also animation looks like you fall down but who cares about animation :D 

4. Golden orbs - wrong location + not working:
Golden orbs should be placed on the bridge and should be hitted to stop the knockback effect.
Don't know for how long tbh. than the effect back.
the effect looks like yellowish rings spinning.

5. Second boss:
The buff you gain (Upwind/Downwind of Altairus) sometimes fall off.
each time the next wind appear and change directions the buff go off and back.
It shouldn't to fall at all only switch between both.

6. NPC packs after 2nd boss (Minister/Exectioner etc) hitting too much damage. (AoE)
Not sure if the damage is legit but we got like 650k damage taken in 10 sec (wipe caused)
in one of the packs before last boss. assuming the average HP is like 75k then that damage is huge.

There is also missing little lighting-triangle on one of the packs you need pull out of it to otherwise spells will be grounded but that is low priority thingy.

**Current behaviour**: ^

**Expected behaviour**: ^

**Steps to reproduce the problem**:

1. 
2. 
3.
